src/components/map/MapboxMap.tsx

```diff
@@ -0,0 +1,172 @@
+import ReactDOMServer from "react-dom/server";
+import React, { useContext, useEffect, useRef, useState } from "react";
+import { AppContext } from "context";
+import { EstimateGradePrevalence, LanguageType } from "types";
+import mapboxgl from "mapbox-gl";
+import httpClient from "httpClient";
+import { getEsriVectorSourceStyle, addEsriLayersFromVectorSourceStyle } from "components/map/EsriMappingUtil";
+import generateSourceFromRecords from "./GeoJsonGenerator";
+import StudyPopup from "components/map/StudyPopup";
+import CountryPopup from 'components/map/CountryPopup'
+import Legend from "components/map/Legend"
+import MapConfig from "components/map/MapConfig"
+import "components/map/Map.css";
+import "components/map/MapboxMap.css";
+import "mapbox-gl/dist/mapbox-gl.css";
+
+mapboxgl.accessToken = process.env.REACT_APP_MAPBOX_API_KEY as string;
```

Comment:
@abeljoseph can you coordinate with @brett-stephen to make sure we have the right API key in place? For prod and staging?

Reply:
nvm should be gud